### DISCUSSION TITLE
Checkbox component doesn't take `form` prop

### DISCUSSION THREAD

**Discussion by vincerubinetti** [adobe/react-spectrum#6262] [General]
Many if not most of the other components accept a `form` attribute, where appropriate, to associate the control with a particular form.

The spec clearly allows controls to be associated with a form by _either_ 1) nesting the controls in the form, _or_ 2) having them separate in the DOM but linked together with `form="some-form-id"` on the control.

I find that component library developers all too often forget about the latter perfectly spec-valid approach. I do not like needing a parent form element because it affects layout, and `display: contents` is still problematic with accessibility in some browsers.

**Comment by vincerubinetti** [comment]
Neither do slider thumbs.

**Reply by vincerubinetti** [reply]
https://github.com/adobe/react-spectrum/issues/4117

**Comment by snowystinger** [comment]
closing as the discussion is on the issue
